ext/enchant/enchant.stubs.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/** @return resource|false */
+function enchant_broker_init() {}
+
+/** @resource $broker */
+function enchant_broker_free($broker): bool {}
+
+/** @resource $broker */
+function enchant_broker_set_dict_path($broker, int $dict_type, string $value): bool {}
+
+/**
+ * @param resource $broker
+ * @return string|false
+ */
+function enchant_broker_get_dict_path($broker, int $dict_type) {}
+
+/**
+ * @param resource $broker
+ * @return resource|false
+ */
+function enchant_broker_request_dict($broker, string $tag) {}
+
+/**
+ * @param resource $broker
+ * @return resource|false
+ */
+function enchant_broker_request_pwl_dict($broker, string $filename) {}
+
+/** @resource $dict */
+function enchant_broker_free_dict($dict): bool {}
+
+/** @resource $broker */
+function enchant_broker_set_ordering($broker, string $tag, string $ordering): bool {}
+
+/** @resource $dict */
+function enchant_dict_quick_check($dict, string $word, array $suggestions = null): bool {}
```

Comment:
`$suggestions` has to be a reference-parameter. While it is an array after the function returns, you can pass anything to the function (and type declarations are only relevant for the latter). The default value is not really `null`. Probably we want something like
```suggestion
function enchant_dict_quick_check($dict, string $word, &$suggestions = UNKNOWN): bool {}
```

Reply:
Oh, I see. The "UNKNOWN" is a little confusing to me but it's done right now!

Reply:
This `UNKNOW` is also a signal to us that something might need to be improved.

Reply:
Nice!